Documentation/gettingstarted/k8s-install-download-release.rst

```diff
@@ -26,3 +21,8 @@
       curl -LO |SCM_ARCHIVE_LINK|
       tar xzf |SCM_ARCHIVE_FILENAME|
       cd |SCM_ARCHIVE_NAME|/install/kubernetes
+
+.. note::
+
+   Make sure you have Helm 3 `installed <https://helm.sh/docs/intro/install/>`_.
+   Helm 2 is `no longer supported <https://helm.sh/blog/helm-v2-deprecation-timeline/>`_.
```

Comment:
Why move this note? It would make sense to me before starting to run `helm`? I don't expect `helm repo add` to break with version 2, but you never know, so let's make sure people use Helm 3 from the start?

Reply:
I actually don't remember why I moved it in the first place 🤔